source/Server.Tests/WorkItemLinkMapperScenarios.cs

```diff
@@ -54,6 +56,7 @@ public void DuplicatesGetIgnored()
             var store = Substitute.For<IJiraConfigurationStore>();
             var jiraClient = Substitute.For<IJiraRestClient>();
             var jiraClientLazy = new Lazy<IJiraRestClient>(() => jiraClient);
+            var log = Substitute.For<ILog>();
```

Comment:
If the tests don't need to use a real `JiraRestClient`, can the changes in this file be reverted?

Reply:
D'Oh, forgot the revert those changes after I changed where we used the logger. Will remove